src/components/MoneyTransactionCreate.js

```diff
@@ -78,7 +79,12 @@ export const MoneyTransactionCreate = ({
         value={formik.values.amount}
       ></TextInput>
       <div className={`${styles.errorMsg}`}>{formik.errors.amount}</div>
-      <Button onClick={() => console.log('transaction created')}>Create</Button>
+      <Button onClick={
+        useCallback(
```

Comment:
If you're defining a useCallback you sadly can't do this inline. Please define a dedicated variable called onMoneyTransactionCreated and define it there